controller/search.js

```diff
@@ -38,38 +36,68 @@ function setup( config, backend, query ){
       return next();
     }
 
-    // backend command
+    logger.debug( '[ES req]', cmd );
+
+    // options for retry
+    // maxRetries is from the ES client which defaults to 3
+    // factor of 1 means that each retry attempt will esclient requestTimeout
+    const operationOptions = {
+      retries: _.get(esclient, 'transport.maxRetries'),
```

Comment:
it might be best to use a different variable name, a variable named `maxRetries` already exists for `elasticsearch-js` with a different meaning.